peerprep/src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import "./App.css";
+import { Button } from "./components/ui/button";
```

Comment:
I would think its easier/more intuitive like this? Since you went to through the trouble of configuring it in `tsconfig.json`.

```suggestion
import { Button } from "@/components/ui/button";
```